survey.html

```diff
@@ -41,12 +41,12 @@ <h1>Survey</h1>
             <textarea name="message"></textarea>
           </label>
           <p>
-            <label for="name" aria-label="required">What is your name?</label>
-            <input type="text" id="name" name="name_field"  />
+            <label for="name">What is your name?<abbr title="required" aria-label="required">*</abbr></label>
+            <input type="text" id="name" name="name_field" required />
```

Comment:
Use an `<abbr>` element to mark the * as per lecture slides
Missing required attribute